### ISSUE TITLE
Feature Request: Integrate sails-connect-mincer

### ISSUE THREAD

**Issue by dapriett** [balderdashy/sails#2956]
Would be nice if Sails had asset pipeline more similar to rails, using something like this: sails-connect-mincer

**Comment by tjwebb** [comment]
sails-connect-mincer looks like a cool project. What's wrong with keeping things modular and letting it remain as an add-on?

**Comment by dapriett** [comment]
Well I was thinking it would be nice if it was the default handler for sails assets, or at least has some official support for it from the sails team.  I haven't been a huge fan of the sails-linker and how it injects the includes in the original files in place.  Mincer seems to be alot cleaner and easily to maintain.

**Comment by tjwebb** [comment]
It's possible, but we already officially support dozens of modules and we have finite resources. We rely quite a bit on the community for stuff like this. If someone would like to start out with doing some work to replace sails-linker with this more powerful alternative, that'd be helpful. We have thousands of feature requests already, and without help it'll be awhile before this makes it to the top of that list.

**Comment by dapriett** [comment]
I created a sails hook which integrates [connect-assets](https://github.com/adunkman/connect-assets) to provide mincer integration.  Been using it within out app, works great: [sails-hook-mincer](https://github.com/dapriett/sails-hook-mincer)

**Comment by sailsbot** [comment]
Thanks for posting, @dapriett.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
